libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPManager.kt

```diff
@@ -104,12 +105,36 @@ internal class IDPManager(
                 codeChallenge: String,
                 onResult: (result: IDPAuthCodeHelper.Result) -> Unit
             ) {
-                IDPAuthCodeHelper.generateAuthCode(WebView(context), userAccount, spConfig, codeChallenge, onResult)
+                val authCodeActivity = context as? IDPAuthCodeActivity
+                if (authCodeActivity == null) {
```

Comment:
Auth code helper expects to be called from IDPAuthCodeActivity,
It gets its web view from the activity and also finishes the activity when done.